### DISCUSSION TITLE
Device objects visible from the GUI but doesn't exist in the Database

### DISCUSSION THREAD

**Discussion by mabdelgaliel** [netbox-community/netbox#13832] [Help Wanted!]
Netbox version: v2.6.6

Everything was working fine for almost 5 years now, but when i did database restore around 1 month ago, i noticed now that there are issues with some device objects, However, Device is visible in the GUI normally with all the details, when i try to delete or perform any action on it, i get "Device.DoesNotExit" error, I get same response when performing the action through API as well.

I did database restore 10s of times before, this is the first time to face this behavior, Sorry, i am running pretty old Netbox version, but only due to the restrictions of the environment it is deployed in.

**Device is present in the GUI:**
![image](https://github.com/netbox-community/netbox/assets/81552708/b2e636d2-5307-4744-990f-b83f30b050d6)

**When trying to delete it, i get below error:**
![image](https://github.com/netbox-community/netbox/assets/81552708/8a204201-af73-4396-8a5b-05d99f463351)

**Comment by abhi1693** [comment]
Are you running a database in active standby mode?

**Reply by mabdelgaliel** [reply]
it is a standalone deployment, i use a script take a copy of the database every hour and move it to a remote server

**Reply by abhi1693** [reply]
This is most likely a database issue. But since you are not running it in ha, I can't say what that specific issue is without inspecting its logs. You should also check the records in the database and what happens when you try to delete.
